### ISSUE TITLE
Categorical data type that sorts by dictionary order?

### ISSUE THREAD

**Issue by nmichaud** [deepankarsharma/perspective#35]
Would be useful for sorting a list of days or months without needing a separate column

**Comment by deepankarsharma** [comment]
So something like an enum?

**Comment by nmichaud** [comment]
yeah, important part is that a pivot on that type has a sort order that might not be alphabetical, and all categories should be in the pivot even if no underlying rows have that value

**Comment by nmichaud** [comment]
Yeah agreed, since each column has it's own vocab. And the way to get a
defined order can be done by insertion order into the vocab, and then using
that for sorting?

**Comment by deepankarsharma** [comment]
Vocabs might go away soon.
